graylog2-server/src/main/java/org/graylog2/grok/GrokPattern.java

```diff
@@ -42,6 +54,18 @@ public String toString() {
                 .toString();
     }
 
+    public static GrokPattern create(String name, String pattern) {
+        return create(null, name, pattern, null);
+    }
+    public static GrokPattern create(ObjectId id, String name, String pattern, String contentPack) {
```

Comment:
This looks as if it should mimic an Auto Value class. So why not make it one in the first place (except for having to adapt the current clients of this class as well)?